api_tests/e2e/rfc003/btc_eth/happy.js

```diff
@@ -28,11 +32,11 @@ const beta_expiry = new Date("2080-06-11T13:00:00Z").getTime() / 1000;
 describe("RFC003: Bitcoin for Ether", () => {
     before(async function() {
         this.timeout(5000);
-        await bitcoin.btc_activate_segwit();
+        await bitcoin.activateSegwit();
```

Comment:
🤡 Since we're not restarting nodes maybe we can just do this once in harness.js when the bitcoin node starts up and remove it from the tests. This would remove a lot of spam from the bitcoin node logs when running multiple tests.

Reply:
look at my latest commit ;)

Reply:
https://github.com/comit-network/comit-rs/pull/807/commits/9fc03e510cd0f7745c6a87089ffb5b28780560af